framework/logging.cpp

```diff
@@ -90,6 +90,10 @@ RtlGetVersion(
 #  include <gnu/libc-version.h>
 #endif
 
+#if __has_include(<openssl/opensslv.h>)
```

Comment:
On some systems (like mine, OpenSUSE), this will be true whether we're linking to OpenSSL or not, because the openssl/ dir is directly in /usr/include:
```
$ ls -l /usr/include/openssl/opensslv.h
-rw-r--r-- 1 root root 4101 Jul  8 05:07 /usr/include/openssl/opensslv.h
```

Instead of `__has_include`, can you use `SANDSTONE_SSL_BUILD` from sandstone_config.h, like you did in sandstone.cpp?